tests/test_session.py

```diff
@@ -0,0 +1,376 @@
+import calendar
+import select
+import socket
+from datetime import datetime
+
+import pytest
+from freezegun import freeze_time
+from lomond import errors, events
+from lomond.session import WebsocketSession
+from lomond.websocket import WebSocket
+from mocket import Mocket, MocketEntry, mocketize
+
+
+@pytest.fixture()
+def session(monkeypatch):
+    monkeypatch.setattr(
+        'os.urandom', lambda len: b'\x00' * len)
```

Comment:
I think this could potentially hide errors, because doing an XOR with 0 doesn't change the result. i.e. `n ^ 0 == n` 

You might want to select a non-zero byte here. Or ideally some pre-determined random bytes.